src/carts/README.md

```diff
@@ -70,7 +70,7 @@ export PATH=$JAVA_HOME/bin:/path/to/maven/bin
 To compile the service:
 
 ```bash
-mvn clean package
+mvn clean package -DskipTests
```

Comment:
Why skip tests ?

Reply:
The unit tests are currently running on port 80 which is a privileged port so running locally will, for many people, raise an error.  They still run in the docker build.  The point of these instructions is to just let the service run standalone so running the tests isn't critical.  The alternative is to allow the port to be configured to something over 1024 (which may be possible already).  I just don't have the cycles to look into it at the moment.